build.gradle.kts

```diff
@@ -148,10 +148,12 @@ configure(libs) {
             dependency("org.aspectj:aspectjweaver:1.9.7")
             dependency("org.assertj:assertj-core:3.20.2")
             dependency("org.codehaus.groovy:groovy-all:2.5.13")
+            dependency("org.grpcmock:grpcmock-junit5:0.5.1")
             dependency("org.freemarker:freemarker:2.3.31")
             dependency("org.jboss.resteasy:resteasy-client:4.7.1.Final")
             dependency("org.mock-server:mockserver-netty:5.11.2")
             dependency("org.mockito:mockito-core:3.12.4")
+            dependency("javax.annotation:javax.annotation-api:1.3.1")
```

Comment:
I have not found any usages for the annotations from this dependency. Is this dependency really needed?

Reply:
This is the standard workaround for JDK 9+ (https://github.com/grpc/grpc-java/issues/3633)
unfortunately [the protoc compiler still uses](https://github.com/grpc/grpc-java/blob/master/compiler/src/java_plugin/cpp/java_generator.cpp#L1204) `javax.annotation.Generated` annotation, but since JDK9 the annotation has been moved to package `javax.annotation.processing.Generated`